packages/core-transaction-pool/src/connection.ts

```diff
@@ -274,29 +274,26 @@ export class Connection implements TransactionPool.IConnection {
                 : undefined;
 
             if (recipientWallet) {
-                transactionHandler.applyToRecipientInPool(transaction, this.walletManager);
+                transactionHandler.applyToRecipient(transaction, this.walletManager);
```

Comment:
Revert. They are purposely named that way because they are only intended to be called by the pool (they get passed the pool wallet manager). Now it's way more ambiguous.

Reply:
After the changes in this patch the `fooInPool()` methods become identical to the `foo()` methods (where `foo` is one of `applyToSender`, `applyToRecipient`, `revertForSender`, `revertForRecipient`):.
```
fooInPool(x, y) {
  foo(x, y);
}
```
which is the *middle man code smell*, so I have fixed it by removing the `fooInPool()` methods had have their callers call just `foo()` instead.

It does not make sense to introduce middle man methods just to indicate where it will be called from. Imagine this:
```
paintItBlack(car) { ... }
paintItBlack(ford)
paintItBlack(audi)
paintItBlack(mazda)

paintItBlackFerrari(car) { paintItBlack(car) }
paintItBlackFerrari(ferrari)
```

Reply:
See other comment.